web/modules/custom/joinup_core/joinup_core.post_update.php

```diff
@@ -25,3 +25,42 @@ function joinup_core_post_update_install_vbo() {
 function joinup_core_post_update_install_email_registration() {
   \Drupal::service('module_installer')->install(['email_registration']);
 }
+
+/**
+ * Move the contact form attachments under the private scheme.
+ */
+function joinup_core_post_update_move_contact_form_attachments() {
+  /** @var \Drupal\Core\File\FileSystemInterface $file_system */
+  $file_system = Drupal::service('file_system');
+
+  $message_storage = \Drupal::entityTypeManager()->getStorage('message');
+  $ids = $message_storage->getQuery()
+    ->condition('template', 'contact_form_submission')
+    ->exists('field_contact_attachment')
+    ->execute();
+
+  foreach ($message_storage->loadMultiple($ids) as $message) {
+    /** @var \Drupal\file\FileInterface $attachment */
+    if ($attachment = $message->field_contact_attachment->entity) {
+      if (empty($attachment)) {
+        continue;
+      }
+      $realpath = \Drupal::service('file_system')->realpath($attachment->getFileUri());
+      if (empty($realpath) || !is_file($realpath)) {
```

Comment:
We've already referred the file system service in the `$file_system` variable. There's no need to call the container again and for each file. Also, from [the PHP official site](http://php.net/manual/en/function.realpath.php):

> realpath() returns FALSE on failure, e.g. if the file does not exist.

I guess we can improve this entire check as:
```php
if (!$attachment || ($file_system->realpath($attachment->getFileUri()) === FALSE)) {
```